validators.py

```diff
@@ -98,6 +98,9 @@ def validate(region, **kwargs):
 
     return validate
 
+    # NI treatment codes do not have Q or L immediately after underscore
```

Comment:
not sure this comment make sense here as it doesn't relate to the treatment code and region lining up in the sample